lib/features/inspection/inspection_view.dart

```diff
@@ -26,7 +26,7 @@ class InspectionBody extends StatelessWidget {
   Widget build(BuildContext context) => SafeArea(
         child: BlocBuilder<InspectionCubit, InspectionState>(
           builder: (_, state) {
-            if (state is InspectionLoaded) {
+            if (state is InspectionLoaded && state.items.length > 1) {
```

Comment:
In cubit, you emitted different state for the case when no booking found, so u don't need to check if this list is empty here, because it's already checked in cubit

Reply:
resolved